include/Symbol.h

```diff
@@ -270,27 +271,29 @@ class ProceduralBlock : public Symbol {
         : children(children), kind(kind), scope(scope) {}
 };
 
-class FormalArgumentSymbol : public Symbol {
+class FormalArgumentSymbol : public VariableSymbol {
```

Comment:
If FAS has no modifiers, shouldn't it have a common base with VariableSymbol, instead of inheriting from VariableSymbol?